docs/pages/includes/cloud/install-linux-cloud.mdx

```diff
@@ -15,7 +15,7 @@
   | sudo tee /etc/apt/sources.list.d/teleport.list > /dev/null
 
   $ sudo apt-get update
-  $ sudo apt-get install teleport-ent
+  $ sudo apt-get install teleport-ent-updater
```

Comment:
Is this package name replacing `teleport-ent` as the way to get the binary, or is it a separate package to provide the updater?